terraform/azure/main.tf

```diff
@@ -41,3 +55,105 @@ resource "azurerm_storage_account" "orsoarpadev" {
     environment = "dev"
   }
 }
+
+resource "azurerm_app_service_plan" "orsoarpadev" {
+  name                = module.app_service_plan_name.result
+  location            = azurerm_resource_group.orsoarpadev.location
+  resource_group_name = azurerm_resource_group.orsoarpadev.name
+
+  sku {
+    tier = "Free"
+    size = "F1"
+  }
+
+  tags = {
+    environment = "dev"
+  }
+}
+
+resource "azurerm_app_service" "orsoarpadev" {
+  name                = module.app_service_name.result
+  location            = azurerm_resource_group.orsoarpadev.location
+  resource_group_name = azurerm_resource_group.orsoarpadev.name
+  app_service_plan_id = azurerm_app_service_plan.orsoarpadev.id
+  https_only = true
+
+  site_config {
+    dotnet_framework_version = "v5.0"
+    scm_type                 = "GitHub"
+    http2_enabled = true
+    health_check_path = "/health"
+  }
+
+  app_settings = {
+    "Logging:IncludeScopes" = "false"
+    "Logging:LogLevel:Default" = "Trace"
+    "Logging:LogLevel:Microsoft" = "Warning"
+    "Logging:LogLevel:Microsoft.Hosting.Lifetime" = "Information"
+    "Logging:LogLevel:Microsoft.EntityFrameworkCore.Database.Command" = "Information"
+    "EmailConfiguration:From" = "dev@arpa.orso.co"
+    "EmailConfiguration:SmtpServer" = "localhost"
+    "EmailConfiguration:Port" = "25"
+    "EmailConfiguration:Username" = ""
```

Comment:
where to get secrets like smtp server passwords from?

Reply:
I am afraid you need to contact the administrator at orso.co to get the username and password that you can use.

Reply:
and pass in the value of the username and password as variables. From GitHub Action, later we can access KeyVault to retrieve them and set it while running terraform.

Reply:
yes I actually know the credentials. what I meant was: how can we get them securely into the configuration :). I added some variables, pls check